File: design-patterns/src/main/java/org/prateek/LLDProject/RideSharingApp/V1/Location.java
package org.Prathamesh.LLDProject.RideSharingApp.V1;

public class Location {
    private double latitude;
    private double longitude;
    public Location(double latitude, double longitude) {
        this.latitude = latitude;
        this.longitude = longitude;
    }
    public double getLatitude() {
        return latitude;
    }

    public double getLongitude() {
        return longitude;
    }
}

File: cpp/LLDProject/RideSharingApp/V1/Location.cpp
#ifndef LOCATION_CPP
#define LOCATION_CPP

#include <string>

class Location {
private:
    double latitude;
    double longitude;

public:
    Location(double lat, double lon) : latitude(lat), longitude(lon) {}

    double getLatitude() const {
        return latitude;
    }

    double getLongitude() const {
        return longitude;
    }
};

#endif 